### ISSUE TITLE
[CLI] Selected runs are not logging media for the key {key_name} but  instead are logging values of type number.

### ISSUE THREAD

**Issue by Godofnothing** [wandb/client#3046]
Description
---
I am running a training loop in `PyTorch` and logging values of the loss and metrics at the end of each epoch.

The way it is done in the code is:

`wandb.log({"epoch" : epoch, **train_stats})`

And the sample of the content of this dict is:

`{'epoch': 0, 'train/loss': 0.9482402317167534, 'train/IoU': 0.0004733342663286095}`

For some reason values of `train/loss` and `train/IoU` are not logged, but instead I have the following picture:

<img width="344" alt="image" src="https://user-images.githubusercontent.com/29793750/146223461-627bdfca-7a58-466c-b30e-8c22f23554ce.png">

`epoch` value is logged normally:

<img width="335" alt="image" src="https://user-images.githubusercontent.com/29793750/146223800-e8328125-e62b-45ea-b854-acf8261d5632.png">

`train/loss` and `train/IoU` are just float values. 

Whan can be the cause of this issue?

**Environment**
- Python Version: 3.7.11
-WandB Version: 0.12.7

**Comment by aidanjdonohue** [comment]
Hi @Godofnothing, can you share a link to the run page so I can take a look?

**Comment by Godofnothing** [comment]
@aidanjdonohue  https://wandb.ai/spiridon_sun_rotator/Unsupervised%20Semantic%20Segmentation?workspace=user-spiridon_sun_rotator - does this link work?

**Comment by Godofnothing** [comment]
@aidanjdonohue  I do not understand, what has happened but running today again this code (without any changes) now produces the expected output the logs.

**Comment by aidanjdonohue** [comment]
Hi @Godofnothing it's possible it was just taking a little while to update the metrics. If you notice this behaviour again, please open this thread back up.
